src/components/AllContents/index.js

```diff
@@ -0,0 +1,114 @@
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+
+import Title from '../Title';
+import InputForm from '../InputForm';
+import YetList from '../YetList';
+import DoneList from '../DoneList';
+
+const Index = () => {
+  const listText = 'list';
+  // 텍스트 이름과 변수 이름을 같게하면 좋을 것 같다는 피드백을 받았던 거 같은데
+  // list 를 다른 변수에 사용해 버렸음. 변수 이름 정하는 센스를 더 길러야 겠음
+
+  const localData = JSON.parse(localStorage.getItem(listText)) || [];
```

Comment:
아니면 따로 변수 선언을 하지 않아도 괜찮을 것 같아요

```suggestion
  // 텍스트 이름과 변수 이름을 같게하면 좋을 것 같다는 피드백을 받았던 거 같은데
  // list 를 다른 변수에 사용해 버렸음. 변수 이름 정하는 센스를 더 길러야 겠음

  const localData = JSON.parse(localStorage.getItem('list')) || [];
```